paasta_tools/cli/cmds/validate.py

```diff
@@ -317,13 +330,33 @@ def path_to_soa_dir_service(service_path):
     return soa_dir, service
 
 
-def validate_tron(service_path):
+def validate_tron(service_path: str, verbose: bool = False) -> bool:
     soa_dir, service = path_to_soa_dir_service(service_path)
     returncode = True
 
     for cluster in list_tron_clusters(service, soa_dir):
         if not validate_tron_namespace(service, cluster, soa_dir):
             returncode = False
+        elif verbose:
+            # service config has been validated and cron schedules are safe to parse
+            service_config = load_tron_service_config(service, cluster)
+            for config in service_config:
+                schedule = config.get_schedule()
+
+                if isinstance(schedule, str) and schedule.startswith("cron"):
+                    config_tz_str = config.get_time_zone() or "US/Pacific"
+                    tz = pytz.timezone(config_tz_str)
+
+                    print(info_message(f"Upcoming runs for {config.get_name()}:"))
+                    next_cron_runs = list_upcoming_runs(
+                        # most cron parsers won't understand our schedule tag, so we need to strip
+                        # that off before passing it to anything else
+                        cron_schedule=schedule.replace("cron", ""),
+                        starting_from=tz.localize(datetime.today()),
+                    )
+
+                    for run in next_cron_runs:
+                        print(f"{run}")
```

Comment:
after looking at the fluffy of the output, what are your thoughts on adding a little whitespace here
```suggestion
                        print(f"'\t{run}")
```